frontend/src/views/profile/Profile.jsx

```diff
@@ -63,53 +63,53 @@ const Profile = ({view = ""}) => {
     return (
         <div className="text-black flex-1 w-10/12 m-auto">
             <div className="inline-block align-top w-4/12 p-10">
-    {
-        user == null ?
-            <div className="w-full mt-5 h-1/2 text-2xl flex justify-center">
-                Loading...
-            </div> :
-            <div className="w-full mt-5">
-                <Avatar className="w-8/12 block m-auto" />
-                <div className="block align-top pt-2 text-center">
-                    <Link to={`/users/${id}/activity`}>
-                        <span className="text-black block font-bold text-xl mt-5">{
-                            user != null ?
-                                user.full_name :
-                                ""
-                        }
-                        </span>
-                    </Link>
-                    <div className='w-9/12 m-auto'>
-                        <Divider />
-                    </div>
-                    <div className="mt-3 flex w-8/12 m-auto">
-                        <div className="w-1/2">
-                            {user.followers.length}
-                            <br />
-                            followers
-                        </div>
-                        <div className="w-1/2">
-                            {user.following.length}
-                            <br />
-                            following
-                        </div>
-                    </div>
-                    {
-                        parseInt(id) === currentUser.current.id ?
-                            <></> :
-                            <button className="btn btn-primary mt-8 w-7/12" disabled={isFollowing == null || isDisabled} onClick={() => handleFollow()}>
-                                {isFollowing ? "Unfollow" : "Follow"}
-                            </button>
-                    }
-                    <Link to={`/users/${id}/history`}><span className="text-blue-700 block mt-5">Learned 20 words</span></Link>
+                {
+                    user == null ?
+                        <div className="w-full mt-5 h-1/2 text-2xl flex justify-center">
+                            Loading...
+                        </div> :
+                        <div className="w-full mt-5">
+                            <Avatar className="w-8/12 block m-auto" />
+                            <div className="block align-top pt-2 text-center">
+                                <Link to={`/users/${id}/activity`}>
+                                    <span className="text-black block font-bold text-xl mt-5">{
+                                        user != null ?
+                                            user.full_name :
+                                            ""

```

Comment:
This is working, can be done using short circuit
> short circuit can cause trouble, use with caution
Ref: https://www.geeksforgeeks.org/javascript-short-circuiting/